packages/canary/src/styles.css

```diff
@@ -97,6 +97,10 @@
     --error: var(--red);
     --warning: var(--orange);
     --ai: var(--purple);
+    --stage: var(--grey-50);
+    --user: var(--grey-40);
+    --git: var(-grey-90);
+    --log: var(--grey-30);
```

Comment:
i see you're not using these variables in the tailwind config. you can delete these,

Reply:
Done.